src/vm/prestub.cpp

```diff
@@ -2359,6 +2360,23 @@ PCODE DynamicHelperFixup(TransitionBlock * pTransitionBlock, TADDR * pCell, DWOR
         }
         pMD->EnsureActive();
         break;
+    case ENCODE_GENERIC_DICTIONARY_ACCESS:
+        {
+            MethodDesc* pContextMD = ZapSig::DecodeMethod(pModule, pInfoModule, pBlob, NULL, &pBlob);
+
+            CORINFO_RESOLVED_TOKEN resolvedToken;
+            resolvedToken.tokenContext = (CORINFO_CONTEXT_HANDLE)pContextMD;
+            resolvedToken.tokenScope = (CORINFO_MODULE_HANDLE)pModule;
+            resolvedToken.token = CorSigUncompressToken(pBlob);
+            resolvedToken.tokenType = (CorInfoTokenKind)CorSigUncompressData(pBlob);
+
+            CEEInfo eeInfo(pContextMD);
```

Comment:
You should fix `ComputeRuntimeLookupForSharedGenericToken` to be a static method and call it directly, without instantiating dummy instance of CEEInfo, to compute the lookup.